CMakeLists.txt

```
@@ -143,7 +143,10 @@ find_package(catkin REQUIRED
   visualization_msgs
 )
 
-find_package(PythonLibs "${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}" REQUIRED)
+option(BUILD_PYTHON_BINDINGS "Build the python bindings using sip" ON)
```

`sip` is only one option to build the python bindings.
```suggestion
option(BUILD_PYTHON_BINDINGS "Build python bindings" ON)
```